src/sources/exec/mod.rs

```diff
@@ -385,6 +388,9 @@ async fn run_command(
             byte_size,
```

Comment:
I don't think this `byte_size` is right either. I believe the stream will return the number of raw bytes where we want to report the number of bytes in-memory. This should be using `BytesSizeOf::size_of`.